### ISSUE TITLE
No Retrieve.php cronjob

### ISSUE THREAD

**Issue by JSdeJong** [jerheij/docker-spotweb#3]
I came a cross your docker file and wanted to try it. It runs very well, and I'm pleased it's dowloading directly from git.

I don't know if it's because I'm trying this docker on UnRaid, but it doesn't seem to run retrieve.php on a regular basis.
I tried to add a cronjob or a script in one of the run-parts locations, but I doesn't seem to run.

Is it possible to add a cronjob to the docker?

**Comment by jerheij** [comment]
> 
> I came a cross your docker file and wanted to try it. It runs very well, and I'm pleased it's dowloading directly from git.
> 
Thanks! I am glad it runs very well for you. I wanted some flexibility in using a specific version of Spotweb so I decided to let it download the code from git in favour of quicker startup.

> 
> I don't know if it's because I'm trying this docker on UnRaid, but it doesn't seem to run retrieve.php on a regular basis.
> I tried to add a cronjob or a script in one of the run-parts locations, but I doesn't seem to run.
> 
> Is it possible to add a cronjob to the docker?
> 
I will have a look and test it myself and post and update once I am done testing. I always ran the retrieve command from the docker host through a cronjob. Don't even exactly know why any more.

**Comment by jerheij** [comment]
I have looked into this and it is quite complicated to have a cron job running on the same container as the application is running on, you would need to run the application process and the crond process to have it run cronjobs properly. I would prefer keeping my container more simple by not having to do that.

I would suggest to look into running a cronjob from your UnRaid OS, I quickly googled it and there seems to be ways to achieve that. I did it the same way, I ran the following command from a cronjob on my docker host:
`docker exec [container] [command]`

I also added a small section to the Docher hub readme about the scheduled retrieve.
